test/README.md

```diff
@@ -0,0 +1,48 @@
+# Usage via command line
+
+To run the test, run the following command from the test directory.
+
+```bash
+python3 -m unittest test_mmd_to_iso.py
```

Comment:
withthe addition of
```python
if __name__=="__main__":
    unittest.main()
```
in the test file, we can remove `-m unittest` from the command line, and having the new test directory -  this line could be changed from:

`python3 -m unittest test_mmd_to_iso.py`

tosomething like:

`python3 test/test_mmd_to_iso.py`